kolibri/plugins/html5_viewer/kolibri_plugin.py

```diff
@@ -7,13 +7,18 @@
 from kolibri.core.content import hooks as content_hooks
 from kolibri.plugins import KolibriPluginBase
 from kolibri.plugins.hooks import register_hook
+from kolibri.utils.conf import OPTIONS
 
 
 class HTML5AppPlugin(KolibriPluginBase):
-    pass
+    kolibri_options = "options"
 
 
 @register_hook
 class HTML5AppAsset(content_hooks.ContentRendererHook):
     bundle_id = "main"
     presets = (format_presets.HTML5_ZIP,)
+
+    @property
+    def plugin_data(self):
+        return {"sandbox": OPTIONS["HTML5"]["SANDBOX"]}
```

Comment:
Perhaps we should give this a more unique/searchable name, e.g. 

```suggestion
        return {"html5_sandbox": OPTIONS["HTML5"]["SANDBOX"]}
```

I'll update the PR and merge

Reply:
I would rather `html5_sandbox_tokens` - also, I think @kollivier is adding another commit to this PR, so would rather not mess with that.

Reply:
Updated.